ansible/roles/config_ocp_cluster/files/tasks/chat-pipelinerun-summary.yml

```diff
@@ -84,7 +84,15 @@ spec:
 
         jq -n --rawfile summary pipelinerun.txt '{"text": $summary}' > payload.json
 
-        echo "Posting message to Slack"
-        curl -s -X POST -H "Content-Type: application/json" \
-          --data @payload.json \
-          "$WEBHOOK_URL"
+        status=$(cat payload.json | grep -oP "Status: (?=\*).\K.*(?=\*)")
+
+        # Skip notifications for operator-hosted-pipeline and passed pipelines
+        if cat payload.json | grep -oP "PipelineRun: *\Koperator-hosted-pipeline" > /dev/null || [ "$status" != "Failed" ];
```

Comment:
Is the `K` a typo in `*\Koperator` or some kind of regex?

Reply:
`\K` ensures that grep returns just `operator-hosted-pipeline` text without the start of line. Otherwise output would contain also `PipelineRun: `